### ISSUE TITLE
[Request] Make attributes work on POCO PropDrawers

### ISSUE THREAD

**Issue by JonInception** [dbrizov/NaughtyAttributes#148]
Hey, loving the plugin :)
One issue I had is trying to add attributes to a serialized (non-mono) class, for instance

```
[System.Serializable]
public class Overrides
{
    [SerializedField] private bool overrideInt;
    [ShowIf(nameof(overrideInt))] // Attribute wont work when nested in mono or SO
    [SerializedField] private int newInt;
}

public class MyMono : MonoBehaviour
{
    [SerializedField] private Overrides overrides;
}
```

Not sure how complex it is, but would be great if attributes would work on the plain C# classes serialized inside a Custom Inspector

**Comment by dbrizov** [comment]
Try [this](https://dbrizov.github.io/na-docs/attributes/special_attributes/allow_nesting.html) and tell me if it works.

**Comment by JonLevin25** [comment]
Nice, didn't know this was supported :)
I can't find that attribute, prob because I'm on version 1.0.4 (waiting on #135 to upgrade)

Is there an alternative for propDrawers on the old version?

**Comment by dbrizov** [comment]
This works in v2.0.0+
I will keep you updated when I create conditional infoboxes - #135
I actually tried to make them. The problem is that in v2.0.0+ the infobox is drawn through a decorator drawer, and not a property drawer. This way I can't get a reference to the target object, and can't get the conditions via reflection. I tried to somehow pass the `this` pointer to the constructor of the infobox attribute, but it's impossible because there is no `this` pointer on the upper code block. I will try to somehow work around it.
